src/lib/content.d.ts

```diff
@@ -561,6 +566,13 @@ interface Image {
 interface VideoAssets {
 	url: string;
 	mimeType: string;
+	fields?: {
+		source?: string;
+		embeddable?: string;
+		height?: string;
+		width?: string;
+		caption?: string;
+	};
```

Comment:
I added these types after refreshing the fixture content and finding them missing